Core/include/Acts/Vertexing/FullBilloirVertexFitter.ipp

```diff
@@ -261,9 +261,7 @@ Acts::FullBilloirVertexFitter<input_track_t, linearizer_t>::fit(
       transMat(5, 3) = 1.;
 
       // cov(V,P)
-      // TODO: This is incorrect (see Ref. (3)), but it will not be needed
-      // anyway once we replace fittedParams with fittedMomentum
-      ActsMatrix<4, 3> covVP = billoirTrack.B;
+      ActsMatrix<4, 3> covVP = -covV * billoirTrack.BCinv;
```

Comment:
Is there a corresponding equation for this in the literature somewhere? It's not obvious to me what's being calculated here, so maybe you could add an extra comment?